docs/snippets/collecting-metrics.md

```diff
@@ -9,6 +9,9 @@ Knative supports different popular tools for collecting metrics:
 
 You can also set up the OpenTelemetry Collector to receive metrics from Knative components and distribute them to other metrics providers that support OpenTelemetry.
 
+    !!! warning
+        You can't use OpenTelemetry Collector and Prometheus at the same time. The default metrics backend is Prometheus. You will need to remove `metrics.backend-destination` and `metrics.request-metrics-backend-destination` keys from the config-observability Configmap to enable Prometheus metrics.
```

Comment:
```suggestion
!!! warning
    You can't use OpenTelemetry Collector and Prometheus at the same time. The default metrics backend is Prometheus. You will need to remove `metrics.backend-destination` and `metrics.request-metrics-backend-destination` keys from the config-observability Configmap to enable Prometheus metrics.
```

Reply:
The indentation is off and doesn't render correctly
![image](https://user-images.githubusercontent.com/1094878/154153301-70a27b5c-9a77-4b00-a06e-a8dfdd0a5911.png)

Reply:
With the suggestion it will look like this
![image](https://user-images.githubusercontent.com/1094878/154153533-67d14251-6009-43c4-bd13-2df87c73da33.png)